tasks/cycles/cycles.js

```diff
@@ -14,24 +14,95 @@ function calculateDiscount(redemption) {
   let discount;
 
   //PLACE YOUR CODE HERE
+  if (redemption >= 0 && redemption <= 350) {
+    discount = 0;
+  } else if (redemption >= 351 && redemption <= 1350) {
+    discount = 15;
+  } else if (redemption >= 1351 && redemption <= 2700) {
+    discount = 30;
+  } else if (redemption >= 2701 && redemption <= 6500) {
+    discount = 45;
+  } else if (redemption >= 6501 && redemption <= 9999) {
+    discount = 60;
+  }
```

Comment:
you do not consider a case when the redemption is a string.

Reply:
Done